pyproject.toml

```diff
@@ -24,6 +24,9 @@ dependencies = [
     "pytest",
     "pytest-cov",
 
+    # required to compile older versions of vyper
+    "vvm",
```

Comment:
maybe this should be part of some optional dependencies? :thinking:

Reply:
i don't think we need to make it optional. vvm is under vyper-controlled supply chain, it is a small dependency, and it has no new external dependencies compared to boa (see https://github.com/vyperlang/vvm/blob/f7b4de84e070062eb4519b2fb5ab0ee8e72269e0/setup.py#L20)